cement/ext/ext_generate.py

```
@@ -26,6 +26,83 @@ class Meta(Controller.Meta):
 
     _meta: Meta  # type: ignore
 
+    def _process_features(self,
+                          features: list[dict[str, Any]],
+                          vars: list[dict[str, Any]],
```

Ruff (0.15.12)

[error] Function argument `vars` is shadowing a Python builtin

(A002)